tools/Documentation/installing-lanraragi/methods.md

```diff
@@ -50,6 +50,10 @@ Ready-to-install packages provided by voluntary maintainers or by a linux distri
 
 {% page-ref page="community.md" %}
 
+## FreeBSD/Jail
+
+Similar to installing from source with an altered process for FreeBSD compatability.
+
```

Comment:
Add a page-ref here so it makes a link in the docs:  

```suggestion
{% page-ref page="jail.md" %}
```

Reply:
Whoops, completely forgot that.